docs/development/issues.md

```diff
@@ -36,41 +36,58 @@ When opening a new issue in the [`electron/electron` issue tracker](https://gith
 will be presented with a template that should be filled in.
 
 ```markdown
-<!--
-Thanks for opening an issue! A few things to keep in mind:
-
-- The issue tracker is only for bugs and feature requests.
-- Before reporting a bug, please try reproducing your issue against
-  the latest version of Electron.
-- If you need general advice, join our Slack: http://atom-slack.herokuapp.com
+<!--  As an open source project with a dedicated but small maintainer team, it can sometimes take a long time for issues to be addressed so please be patient and we will get back to you as soon as we can.
 -->
 
-* Electron version:
-* Operating system:
+### Preflight Checklist
+<!-- Please ensure you've completed the following steps by replacing [ ] with [x]-->
 
-### Expected behavior
+* [ ] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
+* [ ] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
+* [ ] I have searched the issue tracker for an issue that matches the one I want to file, without success.
 
-<!-- What do you think should happen? -->
+### Issue Details
 
-### Actual behavior
+* **Electron Version:**
+  * <!-- (output of `node_modules/.bin/electron --version`) e.g. 4.0.3 -->
+* **Operating System:**
+  * <!-- (Platform and Version) e.g. macOS 10.13.6 / Windows 10 (1803) / Ubuntu 18.04 x64 -->
+* **Last Known Working Electron version:**
+  * <!-- (if applicable) e.g. 3.1.0 -->
 
-<!-- What actually happens? -->
+### Expected Behavior
+<!-- A clear and concise description of what you expected to happen. -->
 
-### How to reproduce
+### Actual Behavior
+<!-- A clear and concise description of what actually happened. -->
 
+### To Reproduce
 <!--
+Your best chance of getting this bug looked at quickly is to provide an example.
+-->
```

Comment:
These paragraphs could probably all go in one `<!--` ... `-->` block to reduce the visual nosiness